### ISSUE TITLE
4-projects not been copied to gcp-projects

### ISSUE THREAD

**Issue by daniel-cit** [GoogleCloudPlatform/terraform-example-foundation-app#33]
## Expected Behavior

When running step 3 in `foundation-extension/4-projects`

```
3. Copy example foundation to new repo cp -RT ../terraform-example-foundation-app/app-foundation/4-projects/ . (modify accordingly based on your current directory).
```
files should be copied.

## Actual Behavior

```
cp: cannot stat '../terraform-example-foundation-app/app-foundation/4-projects/': No such file or directory
```
## Steps to Reproduce the Problem

1. Clone this reposotory
1. Create a new `gcp-projects` folder
2. `cd` into the new `gcp-projects` folder
1. Run `cp -RT ../terraform-example-foundation-app/app-foundation/4-projects/ .`

## Specifications

- Version: d92e89485d2b6ea1caf633b66b41d8a10380e64d
- Platform: Ubuntu 20.04 LTS

**Comment by rutalreja-deloitte** [comment]
@daniel-cit I think you are cloning the repo inside the `terraform-example-foundation-app` folder that is why your path doesn't align. Can you confirm? I think the expectation is the user clones the new `gcp-projects` outside in their root. 
Adressed for 4.5.7 readmes in #36

**Comment by bharathkkb** [comment]
fixed via #36
